hydro_deploy/core/src/hydroflow_crate/build.rs

```diff
@@ -91,6 +91,11 @@ pub async fn build_crate_memoized(params: BuildParams) -> Result<&'static BuildO
                 tokio::task::spawn_blocking(move || {
                     let mut command = Command::new("cargo");
                     command.args(["build"]);
+                    // command.args([
+                    //     "zigbuild".to_string(),
```

Comment:
Yeah, we want to avoid use of `zigbuild`. Using the musl target with `rust-lld` for linking should get us sufficiently far.